terraform/modules/aws-dns-records/README.md

```diff
@@ -0,0 +1,34 @@
+Terraform module: DNS records
+=============================
+
+State: __experimental__
+
+This module creates a set of DNS entries on AWS. As of now it can the following type of records:
+
+* A (`ips`)
+* CNAME (`cnames`)
+
+AWS resources: route53
+
+
+#### How to use the module
+
+```hcl
+module "dns_records" {
+  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws-dns-records?ref=develop"
+  
+  environment = "staging"
+
+  zone_fqdn = "example.com"
+  ips = [ "9.9.9.10", "23.42.23.42" ]
+}
+```
+
+If not further specified, it created entries for the following FQDNs:
+
+* `https.staging.example.com`
+* `ssl.staging.example.com`
```

Comment:
please use the names from the docs; or, if you want different names, we need a discussion and then change those names everywhere in all repositories. For the moment, let's please be consistent.